lib/node_modules/@stdlib/blas/base/drot/docs/repl.txt

```diff
@@ -0,0 +1,112 @@
+
+{{alias}}( N, x, strideX, y, strideY, c, s )
+    Applies a plane rotation.
+
+    The `N` and stride parameters determine how values
+    from `x` and `y` are rotated.
```

Comment:
(1) This line is not properly wrapped at 80 chars and (2) we should update the note.

```suggestion
    The `N` and stride parameters determine how values are accessed at runtime.
```

This may need to be wrapped.